R/source.R

```diff
@@ -5,13 +5,18 @@
 #' @param path Path to files.
 #' @param pattern Regular expression used to filter files.
 #' @param env Environment in which to evaluate code.
+#' @param label If not-`NULL`, will run only test with this label.
```

Comment:
We're always going to match `label` against `desc` in a `test_that()` call.

Should we just call this argument `desc`?

In any case, I think `desc` has to get mentioned here, because "label" isn't an established testthat term.

Reply:
Ooops, I thought that argument was called the label 😭